src/gui/screen_printing.cpp

```diff
@@ -123,6 +123,8 @@ screen_printing_data_t::screen_printing_data_t()
     , w_time_value(this, Rect16(10, 148, 101, 20), is_multiline::no)
     , w_etime_label(this, Rect16(130, 128, 101, 20), is_multiline::no)
     , w_etime_value(this, Rect16(30, 148, 201, 20), is_multiline::no)
+    , g_queue_num(this, Rect16(10, 113, 100, 15), 0, " G-Queue %d", resource_font(IDR_FNT_SMALL))
```

Comment:
visualization of read buffer level shall be refactored/improved in the future

Reply:
We don't want to keep this here. I don't think users care about these internals.